internal/jennies/terraform/models_test.go

```diff
@@ -0,0 +1,45 @@
+package terraform
+
+import (
+	"testing"
+
+	"github.com/grafana/cog/internal/ast"
+	"github.com/grafana/cog/internal/languages"
+	"github.com/grafana/cog/internal/testutils"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRawTypes_Generate(t *testing.T) {
+	test := testutils.GoldenFilesTestSuite[ast.Schema]{
+		TestDataRoot: "../../../testdata/jennies/rawtypes",
+		Name:         "TerraformModels",
```

Comment:
Once you rename the jenny, don't forget to rename it here as well

Reply:
This seems to make less sense. Without the word "terraform", the testdata dir just contains a `Models` directory that doesn't reference Terraform at all.